scripts/gen_vimdoc.py

```diff
@@ -186,6 +186,7 @@
 # Annotations are displayed as line items after API function descriptions.
 annotation_map = {
     'FUNC_API_FAST': '{fast}',
+    'FUNC_API_CHECK_TEXTLOCK': '{check_textlock} |textlock|',
```

Comment:
I don't think the string "check_textlock" is relevant to users, better to just write `not allowed when |textlock| is active`

Reply:
That's exactly. fixed.